docs/use-service.md

```diff
@@ -17,68 +17,85 @@ Once you have your Mapzen API key you'll need include it with Terrain Tile reque
+- `{y}` **vertical position**, counting from the "top", ranges from 0 to variable depending on the zoom
+
+### Tile coordinate resources
+
+- MapTiler.org's [Tiles à la Google Maps: Coordinates, Tile Bounds and Projection](http://www.maptiler.org/google-maps-coordinates-tile-bounds-projection/) has a great visualization that overlays tile coordinates on an interactive map
+- GeoFabrik's [Tile Calculator](http://tools.geofabrik.de/calc/) charts number of tiles per zoom with a customizable bounding box
+
 ##### Terrarium
 
 ```
-https://tile.mapzen.com/mapzen/terrain/v1/terrarium/{z}/{x}/{y}.png?api_key=your-mapzen-api-key
+https://tile.mapzen.com/mapzen/terrain/v2/terrarium/{z}/{x}/{y}.png?api_key=your-mapzen-api-key
 ```
 
 Supported optional tile size variants:
 
-  `https://tile.mapzen.com/mapzen/terrain/v1/{256,512,260,516}/terrarium/{z}/{x}/{y}.png`
+  `https://tile.mapzen.com/mapzen/terrain/v2/{256,512,260,516}/terrarium/{z}/{x}/{y}.png`
 
 The 260 and 516 variants contain 2 pixels of buffering on each edge.
 
 ##### Normal
 
 ```
-https://tile.mapzen.com/mapzen/terrain/v1/normal/{z}/{x}/{y}.png?api_key=your-mapzen-api-key
+https://tile.mapzen.com/mapzen/terrain/v2/normal/{z}/{x}/{y}.png?api_key=your-mapzen-api-key
 ```
 
 Supported optional tile size variants:
 
-  `https://tile.mapzen.com/mapzen/terrain/v1/{256,512,260,516}/normal/{z}/{x}/{y}.png`
+  `https://tile.mapzen.com/mapzen/terrain/v2/{256,512,260,516}/normal/{z}/{x}/{y}.png`
 
 The 260 and 516 variants contain 2 pixels of buffering on each edge.
 
 ##### GeoTIFF
 
 ```
-https://tile.mapzen.com/mapzen/terrain/v1/geotiff/{z}/{x}/{y}.tif?api_key=your-mapzen-api-key
+https://tile.mapzen.com/mapzen/terrain/v2/geotiff/{z}/{x}/{y}.tif?api_key=your-mapzen-api-key
 ```
 
 Note: GeoTIFF format tiles are 512x512 sized so request the parent tile’s coordinate. For instance, if you’re looking for a zoom 14 tile then request the parent tile at zoom 13.
 
 ##### Skadi
 
 ```
-https://tile.mapzen.com/mapzen/terrain/v1/skadi/{N|S}{y}/{N|S}{y}{E|W}{x}.hgt.gz?api_key=your-mapzen-api-key
+https://tile.mapzen.com/mapzen/terrain/v2/skadi/{N|S}{y}/{N|S}{y}{E|W}{x}.hgt.gz?api_key=your-mapzen-api-key
 ```
 
-Note: Skadi files are split into 1° by 1° grids. File names refer to the latitude and longitude of the lower left corner of the tile - e.g. N37W105 has its lower left corner at 37 degrees north latitude and 105 degrees west longitude. For example:  N37W105: `https://tile.mapzen.com/mapzen/terrain/v1/skadi/N37/N37W105.hgt.gz?api_key=your-mapzen-api-key`.
+Note: Skadi files are split into 1° by 1° grids. File names refer to the latitude and longitude of the lower left corner of the tile - e.g. N37W105 has its lower left corner at 37 degrees north latitude and 105 degrees west longitude. For example:  N37W105: `https://tile.mapzen.com/mapzen/terrain/v2/skadi/N37/N37W105.hgt.gz?api_key=your-mapzen-api-key`.
```

Comment:
Technically "v2" as we've been calling it doesn't have breaking changes so I advocate keeping this as `v1` publicly on our Mapzen Terrain Tiles endpoint. Discuss!